target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz7490.dts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz7490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7490";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x20000000>;
```

Comment:
Looking at [this picture](https://boxmatrix.info/w/images/a/ad/FRITZ%21Box_7490_2000-2652_Platine_Frontseite.jpg) the 7490 has 256 MB RAM. The other memory chips are connected to the QCA9558.

Reply:
Yep, thats wrong I will fix it.